src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -286,9 +305,14 @@ public static CloseableHttpClient buildHttpClient(
       @Nullable HttpClientSettingsKey key, File ocspCacheFile, boolean downloadUnCompressed) {
     // set timeout so that we don't wait forever.
     // Setup the default configuration for all requests on this client
-
     int timeToLive = convertSystemPropertyToIntValue(JDBC_TTL, DEFAULT_TTL);
     logger.debug("time to live in connection pooling manager: {}", timeToLive);
+    long connectTimeout = getConnectionTimeout().toMillis();
+    long socketTimeout = getConnectionTimeout().toMillis();
```

Comment:
It is correct that we use the same value for both: connectionTimout and socketTimeout?

Reply:
a mistake introduced in the refactoring - thanks, fixed